src/irr/video/COpenGLFeatureMap.h

```diff
@@ -0,0 +1,1113 @@
+#ifndef __C_OPEN_GL_FEATURE_MAP_INCLUDED__
+#define __C_OPEN_GL_FEATURE_MAP_INCLUDED__
+
+#include <irr\macros.h>
+#include <cstdint>
+
+namespace irr {
+	namespace video
+	{
+
+
+static const char* const OpenGLFeatureStrings[] = {
```

Comment:
move it into the `COpenGLFeatureMap` as a member, so it doesn't pollute the namespace as a global